src/components/MessagesList/MessagesList.vue

```diff
@@ -1113,6 +1116,10 @@ export default {
 		onWindowFocus() {
 			this.refreshReadMarkerPosition()
 		},
+
+		isSystemMessagesGroup(group) {
+			return group.isSystemMessagesGroup ? MessagesSystemGroup : MessagesGroup
+		},
```

Comment:
```suggestion
		messagesGroupComponent(group) {
			return group.isSystemMessagesGroup ? MessagesSystemGroup : MessagesGroup
		},
```

`is` prefix is usually used for boolean flags

Reply:
`MessagesGroup` can now be removed in the `components` options, because it is not used in the template by registration name